src/esp/assets/ResourceManager.cpp

```diff
@@ -884,9 +884,24 @@ bool ResourceManager::loadInstanceMeshData(
 
     for (uint32_t iMesh = start; iMesh <= end; ++iMesh) {
       scene::SceneNode& node = parent->createChild();
-      node.addFeature<gfx::GenericDrawable>(
-          *meshes_[iMesh]->getMagnumGLMesh(), shaderManager_, NO_LIGHT_KEY,
-          PER_VERTEX_OBJECT_ID_MATERIAL_KEY, drawables);
+
+      gfx::Drawable::Flags meshAttributeFlags{};
+      if (meshes_[iMesh]->getMeshData()->hasAttribute(
```

Comment:
@bigbike `meshes_[iMesh]->getMeshData()` returns an optional and GenericInstanceMeshData doesn't currently set that field.  It probably should as there is a `// TODO: this could instead use Mn::Trade::MeshData directly` but just first checking the optional before unwrapping works here.

By the way, you can put python in tests in lldb/gdb by doing something like
```
lldb python
> r -m pytest tests/test_sensors.py -xs
```

(That is how I figured out where the abort was happening/why)

Reply:
Oh, wow. I do not know this trick before. :)
Thank you so much @erikwijmans for looking into this problem and letting me know your method. Certainly I learned something rather useful from you today. I tried immediately. Yes, it gave me where it crashed.

The instanced mesh data is stored independently, and thus the member variable `meshData_` would not be set at this point. It is a trap here.